### PR TITLE
feat: publlic elections tab implemented

### PR DESCRIPTION
**Pull request by Jeetch8**
Resolving issue #545

### CODE REVIEW

File: packages/backend/src/Models/Elections.ts [hunk 1]
```diff
@@ -89,6 +90,7 @@ export default class ElectionsDB implements IElectionStore {
         let query = this._postgresClient
             .selectFrom(tableName)
             .where('head', '=', true)
+            .where("is_public", "=", true)
```
Comment (ArendPeter) [inline packages/backend/src/Models/Elections.ts:93]: Can you remove this one?

getElections will also needs to be able to query private elections

**Review by ArendPeter — CHANGES_REQUESTED** [review]
Thanks for making this change!

I left some minor comments, but overall it looks great

**Comment by Jeetch8** [comment]
Hi Sir, 
thank u for your feedback. I've made the changes. Please take a look.

**Review by ArendPeter — APPROVED** [review]
Looks good, thanks for making the changes!

**Comment by ArendPeter** [comment]
Great work, I merged your PR

Here's the corresponding frontend issue, in case you're interested in taking up another task: https://github.com/Equal-Vote/star-server/issues/471

**Pull request merged**